ui/pages/swaps/swaps.util.js

```diff
@@ -451,6 +451,8 @@ export function getRenderableNetworkFeesForQuote({
   chainId,
   nativeCurrencySymbol,
 }) {
+  const showInFiat = Boolean(nativeCurrencySymbol && conversionRate);
```

Comment:
Something in the swaps changes has caused the middle row to never fill in fee estimate regardless of conversionRate being defined or not: 

![image](https://user-images.githubusercontent.com/4448075/122823356-e007a900-d2a4-11eb-9933-203361f74ea8.png)